packages/beacon-node/src/chain/forkChoice/index.ts

```diff
@@ -21,7 +21,10 @@ import {ChainEventEmitter} from "../emitter.js";
 import {ChainEvent} from "../emitter.js";
 import {GENESIS_SLOT} from "../../constants/index.js";
 
-export type {ForkChoiceOpts};
+export type ForkChoiceOpts = RealForkChoiceOpts & {
```

Comment:
I feel like the use of word "real" is a little unconventional here though I can't come up with a better name. Maybe `RawForkChoiceOpts`?